Example/Tests/Tests.swift

```diff
@@ -1,29 +1,138 @@
-import UIKit
-import XCTest
-import FirebaseHelper
+//import XCTest
+//import FirebaseHelper
+//
+//class Tests: XCTestCase {
+//    func testDecrement() {
+//        let expectation = self.expectation(description: #function)
+//        defer { waitForExpectations(timeout: 10) }
+//
+//        let rand1 = Int(arc4random_uniform(500))
+//        let rand2 = Int(arc4random_uniform(500))
+//        FirebaseHelper.main.set(rand1, at: "test") { (error) in
+//            XCTAssertNil(error, String(describing: error))
+//            let work = GroupWork()
```

Comment:
Am I missing something? It doesn't seem like you've imported GroupWork anywhere.. Haven't tried yet, but I don't think the test would compile correctly.

Reply:
The tests are just placeholder...I think I'll save adding tests for later. I think the correct way to do them would be:

- create a private repo with a firebase plist for a separate firebase project
- write a travis script that:
  - downloads the private repo with credentials in travis env vars
  - copies over the plist into this project somewhere
  - then the tests can login to the firebase and run

Not even sure that would work. Maybe there's an easier way...

Reply:
ahh I get it.. You're having issues because you're trying to test against/hit Firebase. Yeah, what you're suggesting makes sense to me.